var/spack/repos/builtin/packages/r-arrow/package.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class RArrow(RPackage):
+    """'Apache' 'Arrow' <https://arrow.apache.org/> is a cross-language
+    development platform for in-memory data. It specifies a standardized
+    language-independent columnar memory format for flat and hierarchical
+    data, organized for efficient analytic operations on modern
+    hardware. This package provides an interface to the 'Arrow C++'
+    library."""
+
+    cran = "arrow"
+
+    maintainers = ["viniciusvgp"]
```

Comment:
Please use the new `maintainers` API (see https://spack.readthedocs.io/en/latest/packaging_guide.html#maintainers):

```suggestion
    maintainers("viniciusvgp")
```